bin/report_runs.py

```diff
@@ -133,6 +140,15 @@ def report_runs(run_ids, noemail=False):
                         round(clean_yield/1000000000, 1), int(sdata['required_yield']/1000000000),
                         round(mean_cov, 1), sdata['required_coverage']
                     )
+                # Checking for other run elements which are still pending
+                for sample_run_element in query_dict(sdata, 'run_elements') or []:
+                    # Splitting the run element, and generating the run_id by concatenating the first four components
+                    # with an underscore
+                    sample_run_element_sections = sample_run_element.split('_')
+                    sample_run_id = '_'.join(sample_run_element.split('_')[:4])
+                    if query_dict(run_data(sample_run_id), 'aggregated.most_recent_proc.status') == 'processing':
+                        reason += '. Another pending run element already exists'
```

Comment:
I don't know how much sense this message is going to make to the management/lab teams, who don't necessarily know what a run element is. Is there a more generic message we can use?

Reply:
I am not sure about that. @tcezard What do you think?

Reply:
I guess it depends what action we would take.
If this message appears then we (bioinformatics or lab team) will ignore this sample from the repeat list. 
therefore it makes  sense that the sample would be removed from the list altogether, rather than adding a message flagging it.

I would print a warning on the screen highlighting this sample but would remove it from the list.

Reply:
I have just changed this by refactoring it slightly. I've placed the new code in a function, and if a pending run element exists, the current sample is not added to the `sample_repeats` list.